src/class-elements/prod-private-getter-before-super-return-in-constructor.case

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2019 Kubilay Kahveci (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Private getters are installed "when super returns" and no earlier (call in constructor)
+info: |
+  SuperCall: super Arguments
+    1. Let newTarget be GetNewTarget().
+    2. If newTarget is undefined, throw a ReferenceError exception.
+    3. Let func be ? GetSuperConstructor().
+    4. Let argList be ArgumentListEvaluation of Arguments.
+    5. ReturnIfAbrupt(argList).
+    6. Let result be ? Construct(func, argList, newTarget).
+    7. Let thisER be GetThisEnvironment( ).
+    8. Let F be thisER.[[FunctionObject]].
+    9. Assert: F is an ECMAScript function object.
+    10. Perform ? InitializeInstanceElements(result, F).
+
+  EDITOR'S NOTE:
+    Private fields are added to the object one by one, interspersed with
+    evaluation of the initializers, following the construction of the
+    receiver. These semantics allow for a later initializer to refer to
+    a previous private field.
+template: default
+features: [class-methods-private]
+---*/
+
+//- elements
+constructor() {
+    this.f();
+}
+
+//- assertions
+class D extends C {
+    f() { this.#m; }
```

Comment:
nit: Oops, I missed this in the first review, but I think the code style uses 2 space bars.

Reply:
well, this doesn't change the test and there is more than just this to get fixed. 

I'd say we defer style for a follow up or a general fix in the test files.